web-src/src/components/ContainerView.js

```diff
@@ -9,12 +9,170 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+
 import React, { useEffect, useState } from 'react';
-import { Content, Grid, Heading, IllustratedMessage, View } from '@adobe/react-spectrum';
-import WriteIcon from '../icons/WriteIcon';
+import { Grid, View, Tabs, TabList, TabPanels, Item, Text } from '@adobe/react-spectrum';
+import { ToastQueue } from '@react-spectrum/toast'
+
+import AnnotatePen from '@spectrum-icons/workflow/AnnotatePen';
+import Star from '@spectrum-icons/workflow/Star';
+
 import Editor from './Editor';
+import VariationsSection from './VariationsSection';
+import FavoritesSection from './FavoritesSection';
 
 function ContainerView() {
+  const mockVariations = [
```

Comment:
Let's move it out of the function and make it a constant, as it will be recreated at each render.